plugins/outputs/execd/README.md

```diff
@@ -34,8 +34,14 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## Flag to determine whether execd should throw error when part of metrics is unserializable
   ## Setting this to true will skip the unserializable metrics and process the rest of metrics
   ## Setting this to false will throw error when encountering unserializable metrics and none will be processed
+  ## This setting is ignored if use_batch_format is true.
```

Comment:
Why would we ignore this and not do the same behavior of skipping that batch?

Reply:
We have to skip the whole batch if there is a serialization error.

Reply:
Yep, that sounds better. 👍